debian/rules

```diff
@@ -18,10 +18,12 @@ override_dh_auto_configure:
 		-DNEO_OCL_VERSION_MAJOR=$(NEO_MAJOR) -DNEO_OCL_VERSION_MINOR=$(NEO_MINOR) -DNEO_VERSION_BUILD=$(NEO_BUILD) \
 		-DNEO_VERSION_BUILD=${build_id} -DNEO_VERSION_HOTFIX=${build_hf_id} \
 		-DSUPPORT_DG1=TRUE \
-		-DSUPPORT_XE_HP_CORE=TRUE \
 		-DL0_INSTALL_UDEV_RULES=1 \
-		-DNEO_ENABLE_i915_PRELIM_DETECTION=TRUE \
+		-DNEO_ENABLE_I915_PRELIM_DETECTION=TRUE \
 		-DNEO_DISABLE_MITIGATIONS=TRUE \
+		-DNEO_ENABLE_XE_EU_DEBUG_SUPPORT=TRUE \
+		-DNEO_ENABLE_XE_PRELIM_DETECTION=TRUE \
+		-DNEO_XE_PRELIM_HEADERS_DIR=/usr/src/linux-intel-headers-*/include/uapi/drm/ \
```

Comment:
@mckees  what do you think?

Reply:
I'm wondering what happens if multiple -intel headers are installed. Maybe we can do something that grabs the highest version like this? Trying to think if there's a way to make it less ugly though
```
PATH=/usr/src/$(ls /usr/src/ -v | grep "linux-headers.*intel" | tail -1)/include/uapi/drm/
-DNEO_XE_PRELIM_HEADERS_DIR=$PATH
```

Reply:
ya this gets pretty ugly...lets enable this feature in a different PR/step
because we need the enable xe but also we need to make sure this step is fool proof

Reply:
Sounds good. That flag is definitely asking a lot of integration folks. If there's no other way to do it, that might make for a good feature request